monkey/monkey_island/cc/services/reporting/aws_exporter.py

```diff
@@ -6,13 +6,14 @@
 from botocore.exceptions import UnknownServiceError
 
 from common.cloud.aws.aws_instance import AwsInstance
-from monkey_island.cc.environment import EnvironmentConfig
 from monkey_island.cc.services.reporting.exporter import Exporter
 
 __authors__ = ['maor.rayzin', 'shay.nehmad']
 
 logger = logging.getLogger(__name__)
 
+INFECTION_MONKEY_ARN = "324264561773:product/guardicore/aws-infection-monkey"
```

Comment:
@itaymmguardicore is there an option that outside people abuse this?

Reply:
It's just an ID of infection monkey solution in security hub, it's public info
![image](https://user-images.githubusercontent.com/36815064/97959055-37f5ac00-1db7-11eb-8a83-488c76f47842.png)